settings.gradle.kts

```diff
@@ -12,6 +12,15 @@ subdir("entry-generator") {
     include("godot-entry-generator")
 }
 
+// TODO: remove this once the samples are not using the local artifacts
+val ignoreSamples: String? by extra
```

Comment:
@chippmann added a temporary param here. Will be removed the samples are using the actual published artifacts. This is needed for the first time you import the project into IJ!. So before importing you need to run: `./gradlew publishToMavenLocal -PignoreSamples`.

Reply:
ok but you need to add it to the docs as well because there the command is still without the new temporary argument